kitsune/sumo/static/sumo/js/form-wizard.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * A custom element for displaying multi-step forms.
+ *
+ * Uses a named slot and "#activeStep" state to determine which step of the form
+ * should be shown. All child elements where the "name" attribute doesn't match
+ * the "#activeStep" will not be assigned to the named slot.
+ *
+ */
+export class FormWizard extends HTMLElement {
+  #steps = null;
+  #activeStep = null;
+
+  constructor() {
+    super();
+    let shadow = this.attachShadow({ mode: "open" });
```

Comment:
I suppose the open shadow root makes this easier for the automated test. I wonder - @escattone, do we have a way of knowing at runtime whether or not we're in the test environment? If so, we maybe can gain greater encapsulation if we have a closed shadow root here in the non-testing case.

Reply:
@mikeconley @hannajones Hm, I'm not sure what's best. Maybe we could use https://www.npmjs.com/package/detect-mocha or create something similar of our own?

Reply:
I wouldn't block this patch on that, but if you're willing to file a follow-up ticket, it might be worth pursuing just to maximize encapsulation and prevent unnecessary information leakage.